src/Core/Localization/CLDR/Reader.php

```diff
@@ -472,41 +499,73 @@ protected function mapLocaleData(SimpleXMLElement $xmlLocaleData, $supplementalD
                     )
                 ) {
                     $aliasNumSys = $matches[1];
-                    $localeData->currencyPatterns[$numberSystem] = $localeData->currencyPatterns[$aliasNumSys];
+                    $currencyPatterns[$numberSystem] = $localeData->getCurrencyPatterns()[$aliasNumSys];
```

Comment:
Same here, why calling again the getter when you already stored its value in a variable?